docs/examples/MarrmotM14.ipynb

```diff
@@ -2,204 +2,157 @@
  "cells": [
```

Comment:
<p> Having an <code>and</code>  in the text makes it feel like next cell does 2 things while it only initializes. Can we split the Markdown cell or merge the next 2 code cells?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
it is fixed.